src/pages/general/login/actions/spec/loginRequest.spec.ts

```diff
@@ -3,7 +3,7 @@ import ReduxThunk from 'redux-thunk';
 import configureStore from 'redux-mock-store';
 import { FormValidationResult, FieldValidationResult } from 'lc-form-validation';
 import { navigationHelper } from '../../../../../common/helper/navigationHelper';
-import { loginApi } from '../../../../../rest-api/login/loginAPI';
+import * as loginApi from '../../../../../rest-api/login';
```

Comment:
login is the folder, we are using "index.ts" approach, by using this you can later on refactor inners without affecting the outer world, the loginApi, is true, as an alias it can be a bit dangerous but on the other hand "login" is too ambiguous